crates/cli/Cargo.toml

```diff
@@ -20,8 +20,9 @@ anyhow = { version = "1.0.70" }
 clap = { version = "4.1.13", features = ["derive"] }
 indicatif = { version = "0.17.5" }
 # Do *NOT* upgrade beyond 1.0.171 until https://github.com/serde-rs/serde/issues/2538 is fixed
```

Comment:
Is this still important because `biome` depends on `serde` version `1.0.217`

Reply:
Let's avoid changing this in the same PR if we can.

Reply:
ok then can i remove that comment to avoid confusion later?

Reply:
Yes please do.